sw/host/opentitanlib/src/app/config/opentitan_qemu.json

```diff
@@ -0,0 +1,46 @@
+{
+  "includes": ["/__builtin__/opentitan.json"],
+  "interface": "qemu",
+  "pins": [
```

Comment:
I am guessing that those definitions are just to make opentitantool happy and that QEMU OT does not support gpios yet?

Reply:
That's what I added them for, although now I see that I can remove them